websites/S/Stats.fm/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "2Lost4Discord",
+		"id": "421991668556759042"
+	},
+	"contributors": [
+		{
+			"name": "BigBoi©",

```

Comment:
A person isn't copyrighted. Also did they contribute to a good part of the presence's code?
```suggestion
			"name": "BigBoi",

```

Reply:
Please answer this

Reply:
That is my discord username for gods sake

Reply:
Ohh sorry i forgot to answer that. Yes that BigBoi-TM and yes he contributed in a good part, he helped me showing stuff and solving problems

Reply:
Pretty sure you need much more than that.
![image](https://user-images.githubusercontent.com/77577746/168000983-08e80f82-df65-445c-a58c-766f4660e21a.png)

Reply:
If you do consider he contributed to a good amount of the presence that’s fine, but no you cannot add that back because it doesn’t make sense and might confuse others

Reply:
Like I've said before, I've had no trouble with this sort of thing as I have another presence with it AND it's on your store:

![image](https://user-images.githubusercontent.com/96930330/168039698-812efd1e-5568-422f-98ff-08fbe9d900f4.png)

Reply:
Hey, just wanted to ask but as he already said he already created a presence with the © in the name, so i just wanted to ask what the status of that is. Can i add it back or can i not add it back?

Reply:
If that went through then it must've been a mistake on our end, you cannot put that symbol in the name of a contributor, please keep it removed. The other presence will be updated shortly to remove that

Reply:
> If that went through then it must've been a mistake on our end, you cannot put that symbol in the name of a contributor, please keep it removed. The other presence will be updated shortly to remove that

Alright, so now everything is fine with the presence or is there anything else i have to add?